maintenance/wikia/jwplayermigration/migrateWikiFactoryVariableToService.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Add JWPlayer mediaId to wgFeaturedVideo
+ *
+ */
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+
+class MigrateOoyalaVideos extends Maintenance {
+
+	protected $dryRun = true;
+
+	protected $map = [];
+
+	// pass access token
+	const ACCESS_TOKEN = '';
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Migrate videos";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+
+		$videos = array_map( 'str_getcsv', file( 'output.csv' ) );
+		$map = [];
+
+		foreach ( $videos as $video ) {
+			$map[$video[1]] = $video[0];
+		}
+
+		$this->map = $map;
+	}
+
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( 'dry-run' );
+		$this->output( "Start \n" );
+
+		$this->doUpdate( 1689, 'wgArticleVideoFeaturedVideos' );
+		$this->doUpdate( 1749, 'wgArticleVideoFeaturedVideos2' );
+	}
+
+	private function toDevURL($url) {
+		return str_replace('.wikia.com', '.ktatala.wikia-dev.pl', $url);
+	}
+
+	private function addMapping($url, $mediaId) {
+		global $wgDevelEnvironment;
+
+		$data = [
+			'url' => $url,
+			'media_id' => $mediaId,
+		];
+		$options = [
+			'postData' => json_encode( $data ),
+			'headers' => [
+				'Content-Type' => 'application/json',
+				'Accept' => 'application/json',
+				'X-Wikia-AccessToken' => self::ACCESS_TOKEN,
```

Comment:
if you'd like I can easily create internal endpoint for creating mapping in the service so there is no need to use access token